### PR TITLE
Update Pages “home”

### PR DESCRIPTION
**Pull request by liz-shinn**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by liz-shinn** [comment]
@ironng Ready for review! :) This adds 3 new team members, updates everyone's headshots, and adds the right website for DHVC since they'd updated it (https://github.com/keep-network/website/issues/183)

File: src/pages/index.md [hunk 1]
```diff
@@ -3,124 +3,136 @@ template: home-page
...
       social_links:
-        - url: https://twitter.com/dmitry_paremski
-        - url: https://www.linkedin.com/in/paremski/
-        - url: https://github.com/dimpar
-        - url: http://keybase.io/dparemski
-    - name: Liam Zebedee
+        - url: https://twitter.com/ngrinkevich
+        - url: https://www.linkedin.com/in/nikgrinkevich
+        - url: https://github.com/ngrinkevich
+        - url: https://keybase.io/nikgrinkevich
```
Comment (Shadowfiend) [inline src/pages/index.md:122]: Nik's moved on to Fold-land :)

**Comment by Shadowfiend** [comment]
I can take this one @ironng :)

File: src/pages/index.md (hunk 1 above)
Comment (liz-shinn) [reply src/pages/index.md:122]: 👍 Will fix this!

**Pull request merged**